### PR TITLE
add Lazy_frame.profile

### PR DESCRIPTION
**Pull request by somilgo**

### CODE REVIEW

**Comment by mt-caret** [comment]
@somilgo could you re-merge upstream changes into your repo, since ? (Also, feel free to create a branch in this repo from next time on, and I can do that sort of thing on my side as well)

File: rust/src/lazy_frame.rs [hunk 1]
```diff
@@ -86,6 +86,16 @@ ocaml_export! {
         .to_ocaml(cr)
     }
 
+    fn rust_lazy_frame_profile(cr, lazy_frame: OCamlRef<DynBox<LazyFrame>>)-> OCaml<Result<(DynBox<DataFrame>, DynBox<DataFrame>), String>> {
```
Comment (mt-caret) [inline rust/src/lazy_frame.rs:89]: Have we formatted this code (OCaml as well)? Seems off.

**Review by mt-caret — CHANGES_REQUESTED** [review]
LGTM once we fix the formatting.

**Comment by somilgo** [comment]
Whoops, I opened this pull request with my master branch. Closing and creating a new PR with my branch specific to this PR.

**Pull request closed without merging**